manifests/libvirt.yaml.in

```diff
@@ -59,6 +61,9 @@ spec:
       - name: host-sys
         hostPath:
           path: /sys
+      - name: host-proc
+        hostPath:
+          path: /proc
```

Comment:
I would expect the changes to this file to be part of the commit which actually implements the proposal, rather than part of the design docs commit.